README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
+* [Support for the major data processing technologies](./docs/utils.md) — [Dataproc](https://cloud.google.com/dataproc),
```

Comment:
What will be in utils.md?

Reply:
```
1. Utils
    1. BigQuery
    1. Apache Beam
    1. Dataproc
    1. Job monitoring
```

Reply:
So maybe instead utils better name would be `processing`? And another section dedicated for `monitoring`?

Reply:
done